tests/temp_run_tests.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -x
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)